test/xbgpu/test_engine.py

```diff
@@ -1028,6 +1028,10 @@ async def test_engine_end_to_end(
         non-contiguous batch indices (i.e. missing Chunks in the receiver), there
         is an artificial gap introduced in the data sent to/processed by the
         XBEngine.
+
+        Lastly, more complete testing of `?beam` requests is carried out by
```

Comment:
"More" complete relative to what?

Reply:
Relative to how it was being tested before. Although I see how looking at the test as-is doesn't really give context of how it was (not the point of it either).

I'll reword.